### PR TITLE
CHK-425: Replace address-form by checkout-shipping components

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What problem is this solving?

Replace `@vtex/address-form` components by `vtex.checkout-shipping`, so we have a uniform experience for the customer.

#### How to test it?

[Workspace](https://lucas--checkoutio.myvtex.com/cart)

#### Describe alternatives you've considered, if any.

N/A

#### Related to / Depends on

vtex-apps/checkout-shipping#14

### CODE REVIEW

**Comment by lucasecdb** [comment]
tests are failling because of the typings in `package.json`, but they are passing and will pass on CI once vtex-apps/checkout-shipping#14 is deployed

File: react/package.json [hunk 1]
```diff
@@ -21,20 +21,25 @@
...
     "@vtex/country-codes": "^1.0.2",
     "@vtex/styleguide": "^9.95.0",
     "@vtex/test-tools": "^3.0.2",
     "apollo-client": "^2.5.1",
-    "typescript": "3.7.3",
-    "vtex.address-form": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.address-form@4.4.3/public/_types/react",
+    "typescript": "3.9.7",
+    "vtex.address-context": "https://lucas--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.address-context@0.3.0+build1603978926/public/@types/vtex.address-context",
+    "vtex.address-form": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.address-form@4.6.3/public/_types/react",
+    "vtex.checkout-components": "https://lucas--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.checkout-components@0.4.0+build1603997531/public/@types/vtex.checkout-components",
```
Comment (jeffersontuc) [inline react/package.json]: Do you pretend to change this?

Comment (jeffersontuc) [reply react/package.json]: Nevermind... I just saw you comment above.

**Comment by wdsrocha** [comment]
LGTM, but there are some [linting warnings](https://github.com/vtex-apps/shipping-calculator/pull/41/checks?check_run_id=1349620790). Will you fix them?

**Review by jeffersontuc — DISMISSED** [review]

**Comment by lucasecdb** [comment]
> LGTM, but there are some [linting warnings](https://github.com/vtex-apps/shipping-calculator/pull/41/checks?check_run_id=1349620790). Will you fix them?

I fixed some of them in 9e0b185, but must of them are either on mock files or type declarations files.. I'll leave this as is for now, but we can try to fix them later.

**Review by alvim — APPROVED** [review]

**Review by wdsrocha — APPROVED** [review]
:worry-neat-ok:

**Pull request merged**